source/process/release-metrics-definitions.rst

```diff
@@ -0,0 +1,136 @@
++-----------------------------------------+------------------------------------------------------------------------------+
+| Community + customer bugs reported      | Use "Customer-bug" and "Community-bug" label for release month, from         |
+| during release timeframe (17th to 16th) | 17th to 16th (from previous release day to current release day).             |
++-----------------------------------------+------------------------------------------------------------------------------+
+| Number of customer bugs fixed           | "Customer-bug" tickets closed and marked for current release.                |
+| during release                          |                                                                              |
++-----------------------------------------+------------------------------------------------------------------------------+
+| Total valid bugs in fix version         | After closing current release:                                               |
+|                                         |                                                                              |
+|                                         | project = Mattermost AND issuetype = Bug AND resolution not in (Duplicate,   |                                         
+|                                         | "Cannot Reproduce", "Won't Fix") AND fixVersion = latestReleasedVersion()    |
++-----------------------------------------+------------------------------------------------------------------------------+
+| Total valid bugs in fix version found   | Check "Se", "Selenium-found, "Rainforest-found" Jira labels.                 |
+| by test automation                      |                                                                              |
++-----------------------------------------+------------------------------------------------------------------------------+
+| Total valid bugs found after RC1 is cut | 1. Check Jira timezone + Pre-release timezone and make sure times match      |
+|                                         | 2. Replace START with date (yyyy-MM-dd HH:mm) RC1 was cut                    |
+|                                         | 3. Replace END with date (yyyy-MM-dd HH:mm) test servers returned to master  |
+|                                         |                                                                              |
+|                                         | project = Mattermost AND issuetype = Bug AND resolution not in (Duplicate,   |
+|                                         | "Cannot Reproduce", "Won't fix") AND created > "START" AND created < "END"   |
++-----------------------------------------+------------------------------------------------------------------------------+
+| Valid bugs found after RC1 fixed in     | After closing current release, & adjust dates as per above:                  |
```

Comment:
For this and others where the data is received from Jira, propose mentioning it somewhere, e.g. `After closing current release, adjust dates as per above, and use this Jira query:`

(You have referenced Jira in some, but it's helpful to reference in all where appropriate, so that it's obvious to the reader)